tests/test_package.py

```diff
@@ -60,10 +60,16 @@ def test_share_git_package():
 
 
 def test_load_shared_git_package():
-    git_package = rh.package(name="@/shared_git_package")
+    git_package = rh.Package.from_name(name="@/shared_git_package")
     assert git_package.config_for_rns
 
 
+def test_cuda_version_install_for_torch():
```

Comment:
This is a case where I think we need to unit test a longer list of install_cmds and the expected command that would be run. There are a bunch of ways users might specify the install string that we already know of. I wouldn't call install, just test the fn which generates the command.